File: app/src/main/java/org/apache/fineract/ui/online/identification/createidentification/OnNavigationBarListener.java
package org.apache.fineract.ui.online.identification.createidentification;


import org.apache.fineract.data.models.customer.identification.Identification;

/**
 * @author Rajan Maurya
 *         On 17/07/17.
 */
public interface OnNavigationBarListener {

    interface IdentificationCard {
        void setIdentificationDetails(Identification identificationDetails);
    }
}

File: app/src/main/java/org/apache/fineract/ui/online/accounting/ledgers/OnNavigationBarListener.kt
package org.apache.fineract.ui.online.accounting.ledgers

import org.apache.fineract.data.models.accounts.Ledger

interface OnNavigationBarListener {

    interface LedgerDetails {
        fun setLedgerDetails(
                type: String,
                identifier: String,
                name: String,
                description: String,
                showAccountInChart: Boolean
        )
    }

    interface LedgerReview {
        fun getLedger(): Ledger?
    }

    interface SubLedger {
        fun setSubLedger(subLedgerList: List<Ledger>)
    }
}